Library/NSBundleType.swift

```diff
@@ -27,6 +30,14 @@ extension NSBundleType {
     return self.infoDictionary?["CFBundleVersion"] as? String ?? "0"
   }
 
+  public var isTest: Bool {
```

Comment:
Ah interesting, what's this used for?

Reply:
When I changed this line: https://github.com/kickstarter/ios-oss/pull/385/files#diff-63e4e32ea3a28f8587151aa70d4525fbR125 we had a snapshot test that started failing because the debug tools button was being shown on test builds. I added this so we can hide the debug menu on test bundleIds.